meilisearch-core/src/update/documents_deletion.rs

```diff
@@ -180,5 +180,9 @@ pub fn apply_documents_deletion(
 
     compute_short_prefixes(writer, &words, index)?;
 
+    // update is finished; update sorted document id cache with new state
+    let mut document_ids = index.main.internal_docids(writer)?.to_vec();
+    super::cache_documents_document_ids_sorted(writer, &ranked_map, index, &mut document_ids)?;
```

Comment:
Should the `document_ids` here be the whole list of all documents ids of the database?
If so, are you sure this is the case? Isn't this list only the documents ids related to this specific update?

Reply:
there may be documents in the database that are not in the update right? I need to sort them too with the new documents